aspects/aspects.go

```diff
@@ -86,6 +86,35 @@ func notFoundErrorFrom(a *Aspect, request, errMsg string) *NotFoundError {
 	}
 }
 
+type BadRequestError struct {
+	Account    string
+	BundleName string
+	Aspect     string
+	Operation  string
+	Request    string
+	Cause      string
+}
+
+func (e *BadRequestError) Error() string {
+	return fmt.Sprintf("cannot %s %q in aspect %s/%s/%s: %s", e.Operation, e.Request, e.Account, e.BundleName, e.Aspect, e.Cause)
+}
+
+func (e *BadRequestError) Is(err error) bool {
+	_, ok := err.(*BadRequestError)
+	return ok
+}
+
+func badRequestErrorFrom(a *Aspect, operation, request, errMsg string, v ...any) *BadRequestError {
```

Comment:
We use `interface{}` everywhere else, rather than `any`. Although at some point, swapping might be good. The stdlib uses `any` nowadays: https://cs.opensource.google/go/go/+/refs/tags/go1.21.5:src/encoding/json/encode.go;l=158